docs/integrations/sources/surveymonkey.md

```diff
@@ -1,48 +1,64 @@
 # SurveyMonkey
 
-This page guides you through the process of setting up the SurveyMonkey source connector.
+Learn how to set up the SurveyMonkey source connector. A source connector extracts data from SurveyMonkey and transfers the data to a destination. Learn more about connectors [here](/using-airbyte/core-concepts/).
 
 :::note
 
-OAuth for Survey Monkey is officially supported only for the US. We are testing how to enable it in the EU at the moment. If you run into any issues, please [reach out to us](mailto:product@airbyte.io) so we can promptly assist you.
+OAuth for SurveyMonkey is available in the US only. We are testing how to enable OAuth in the EU. [Reach out to us](mailto:product@airbyte.io) if you run into any issues with authentication.
 
 :::
 
 <!-- env:oss -->
-## Prerequisites
+## Know before you begin
 
-**For Airbyte Open Source:**
-
-* Access Token
+You need a SurveyMonkey access token if you're using Airbyte Open Source to set up the connector. Use this token to authenticate your SurveyMonkey account with Airbyte Open Source.
 <!-- /env:oss -->
 
-## Setup guide
+## Set up the SurveyMonkey connector in Airbyte
+
 ### Step 1: Set up SurveyMonkey
-Please read this [docs](https://developer.surveymonkey.com/api/v3/#getting-started). Register your application [here](https://developer.surveymonkey.com/apps/) Then go to Settings and copy your access token
+
+1. Create or log in to your [SurveyMonkey account](https://developer.surveymonkey.com/apps/).
```

Comment:
1. Completely rewrote this step. There needs to be a detailed list of steps to orient the user to SurveyMonkey's UI. I created a SurveyMonkey account to verify the steps myself.

2. The main point of this step is to register a new app and get the access token. That should be the first focus.

3. Reading the SurveyMonkey docs should be optional and not a required first step.